src/components/CollapsibleSection/index.tsx

```diff
@@ -13,9 +14,18 @@ import Collapsible from './Collapsible';
 type CollapsibleSectionProps = ChildrenProps & {
     /** Title of the Collapsible section */
     title: string;
+
+    /** Style of title of the collapsible section */
+    titleStyle?: StyleProp<TextStyle>;
+
+    /** Any additional styles to apply */
+    wrapperStyle?: StyleProp<ViewStyle>;
+
+    /** Whether or not to show border between section title and exandable items */
+    shouldShowSectionBorder?: boolean;
```

Comment:
I am not sure where this part of the design is discussed. But I checked the terms step of payment and the other integrations.

<img width="395" alt="image" src="https://github.com/Expensify/App/assets/3069065/4391aae1-b0a1-4d19-8af1-0823473e0500">


<img width="832" alt="image" src="https://github.com/Expensify/App/assets/3069065/373cd094-3808-4f53-8d43-7fda4a4c215c">

Reply:
I am not sure about it. I was curious why the property was added if it was not required for the integrations PR. We added for Terms page, which looks unrelated to the current PR.

Reply:
@SzymczakJ ^